src/app/shipyard/Module.js

```diff
@@ -135,14 +129,93 @@ export default class Module {
    * @param {Number} modified Whether to return the raw or modified value
    * @return {Number} The value queried
    */
-  _getValue(name, modified) {
+  get(name, modified = true) {
```

Comment:
not really a fan of setting args like that.

```
if (!modified) {
modified = true;
}

Reply:
Uhm. Is a matter of preference but is less statements :)